### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by strugee** [straticjs/RFCs#22]
Maybe the JS logo's yellow background with a `|` character and a file? I really don't know. I also like how "Stratic" kinda sounds like "stratosphere", but...

**Comment by strugee** [comment]
Alrighty, I'm attaching some drafts:

![stratic-logo-asteroid](https://user-images.githubusercontent.com/911174/34466572-52947368-ee8d-11e7-91a7-5d1e78015c3f.jpg)
![stratic-logo-rocket](https://user-images.githubusercontent.com/911174/34466573-580d4892-ee8d-11e7-8c6e-987f5ee03789.jpg)
![stratic-logo-rocket-earth-bottom-right](https://user-images.githubusercontent.com/911174/34466574-5997bf26-ee8d-11e7-9527-0504c66042c4.jpg)

There are also variants of these three with pipes added in; these are shamelessly stolen from the [WHATWG streams spec logo](https://github.com/whatwg/whatwg.org/blob/master/resources.whatwg.org/logo-streams.svg), cause a streams spec seemed fitting (and also the pipe was really well done):

![stratic-logo-asteroid-with-pipe](https://user-images.githubusercontent.com/911174/34466583-8e041aa2-ee8d-11e7-9660-fc13de5e36f4.jpg)
![stratic-logo-rocket-with-pipe](https://user-images.githubusercontent.com/911174/34466585-91c83416-ee8d-11e7-93ff-6cef42045b00.jpg)
![stratic-logo-rocket-earth-bottom-right-with-pipe](https://user-images.githubusercontent.com/911174/34466586-9314fff2-ee8d-11e7-8964-d6e2a1ee7524.jpg)

Thoughts?

**Comment by strugee** [comment]
I'm definitely leaning towards the pipe variants at this point, but could be convinced otherwise. I might also mock up versions with "StraticJS" written underneath.

**Comment by Riven-Spell** [comment]
I prefer the pipe variants. Perhaps rather than using a mario-esque pipe, try and use something like the typical pipe character, |.

**Comment by strugee** [comment]
Yeah, I thought about using `|` in some way (see the description) but eventually rejected it because I don't think it would be clear that it was a pipe as opposed to just a random line.

**Comment by strugee** [comment]
A friend suggests this design:

![stratic-logo-blank-file-with-pipe svg](https://user-images.githubusercontent.com/911174/34467410-d63503b4-eea5-11e7-9483-9da7233b83b5.png)

it _does_ seem to be cleaner which is a bonus, but I worry that it's too generic. Seems like it could be the logo for any old gulp project, or even gulp itself.

**Comment by strugee** [comment]
Original SVGs and related assets are here if anyone wants them: https://cloud.strugee.net/index.php/s/31PYqHOUo9NF5Ur
